Formula/nfdump.rb

```diff
@@ -11,8 +11,16 @@ class Nfdump < Formula
     sha256 "5bbd8ec9a51cde3c6ed0aaeb1b77a4f2ba59bea99d5304fa1415a7e2e0d5280b" => :mountain_lion
   end
 
+  option "with-readpcap", "Read netflow packets from a give pcap_file instead of the network. Is intended for debugging only."
```

Comment:
The only "debugging only" thing I see in the man page is `-X`, not `-f`. Are you sure it's relevant here?

```
       -X Compiles the filer syntax and dumps the filter engine table to stdout.  This is for  debug-
          ging purpose only.
```

Either way, I'd go for brevity. The user can read the man page for details.